content/manager/user-management.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: bt_wiki
+title: Managing Users
+category: Manager
+draft: false
+weight: 1400
+---
+
+
+In Cloudify, you can connect to your LDAP or AD system to integrate users and user groups, or you can add users and user groups individually. You must select to either use the LDAP system or to add users and groups individually. You cannot do both. User management is performed using the CLI.
+
+{{% gsTip title="User Management Credentials" %}}
+You must have Cloudify Manager administrator permissions to perform user-management related actions.
+{{% /gsTip %}}
+
+## Integrating with an LDAP System
+If you integrate LDAP or your AD into Cloudify Manager, you use the users and user groups defined there for access to the Manager. 
+
+To connect Cloudify Manager with LDAP/AD, you must know the the URL of the service and have sufficient credentials to perform searches and so on.
+
+You configure Cloudify with the LDAP configuration during the bootstrap process, in the `manager-input` section. You can also use the API to configure an LDAP connection after Cloudify Manager is installed, using the `cfy ldap set` command, as long as the Manager is clean, meaning that no tenants, groups, users or resources exist in it.
+
+### How Cloudify Manager Works with the LDAP/AD Service
+
+When a user logs in to Cloudify Manager, their credentials are passed to the LDAP/AD service for authentication. By default, all users in the LDAP/AD service are authenticated to Cloudify Manager, however only users who have specific permissions for a tenant can access it.
+
+When a user logs into Cloudify Manager, the service authenticates the user and returns a list of any groups to which the user belongs.
+
+In Cloudify Manager, if you have added a group to a tenant, using the process that complies with the requirements for defining a group specified in LDAP/AD, all users in that LDAP group can access the tenant (unless an individual user is specifically suspended from the tenant.) The For more information about specifying LDAP/AD-compliant user-group names, see Adding Users on the Tenant Management page.
```

Comment:
We don't currently have the option of suspending a specific user from a tenant, we can remove this comment.